sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java

```diff
@@ -81,7 +81,7 @@
 
     private Map<DefaultAppCenterFuture<?>, ServiceCall> mPendingCalls = new HashMap<>();
 
-    private final Map<String, ServiceCall> mOutgoingPendingOperationCalls = new HashMap<>();
+    private final HashMap<String, ServiceCall> mOutgoingPendingOperationCalls = new HashMap<>();
```

Comment:
should not put null value on Map as its implementation dependent so we get a lint warning. By telling we depend on this implementation we can depend on ability to set null values

Reply:
+1.

Plus `mOutgoingPendingOperationCalls` is only a private property so it's not leaking anything and there's no point to use `Map` here.

Reply:
That was my feedback to use Map as this is the pattern to use interface whenever possible. This is an exception.